.coderabbit.yaml

```
@@ -0,0 +1,57 @@
+  # Review level settings
+  review_status: "all_files"
+  
+  # Enable high-level summary
+  high_level_summary: true
+  
+  # Enable poem
+  poem: false
+  
+  # Enable review details
+  review_details_enabled: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)